detox/src/ios/expect.js

```diff
@@ -58,9 +58,13 @@ class TapAtPointAction extends Action {
 }
 
 class LongPressAction extends Action {
-  constructor() {
+  constructor(duration) {
     super();
-    this._call = invoke.callDirectly(GreyActions.actionForLongPress());
+    if (duration) {
```

Comment:
let's add default value, drop the condition, and use only `GreyActions.actionForLongPressWithDuration`